UI/Selenium/Steps/BookingListSteps.cs

```diff
@@ -140,21 +140,19 @@ public void WhenTheVHOScrollsToTheHearing()
             Stopwatch timer = new Stopwatch();
             timer.Start();
             var isFound = false;
-            while (!isFound && timer.Elapsed <=TimeSpan.FromSeconds(Config.DefaultElementWait))
+            Actions actions = new Actions(Driver);
+            int attempts = 0;
+            while (!isFound && attempts <= 20)
             {
-                var allHearings = Driver.FindElements(BookingListPage.AllHearings);
-                var lastHearingElement = allHearings[allHearings.Count-1];
-                Actions actions = new Actions(Driver);
-                actions.MoveToElement(lastHearingElement);
-                actions.Perform();
-                var element = ExtensionMethods.FindElementWithWait(Driver, BookingListPage.HearingSelectionSpecificRow(_hearing.Case.CaseNumber), _scenarioContext);
-                if (element!=null)
+                isFound = Driver.FindElement(OpenQA.Selenium.By.TagName("body")).Text.Contains(_hearing.Case.CaseNumber);
+                if (!isFound)
                 {
                     actions = new Actions(Driver);
-                    actions.MoveToElement(element);
-                    actions.Perform();
-                    isFound=true;
+                    actions.SendKeys(OpenQA.Selenium.Keys.End).Perform();
```

Comment:
Does this work in all cases?  I recall in a previous project that it would not, it became necessary to do a browser scroll until the end of the page was reached.

Reply:
Should do @MichaelKeaneathmcts. And yes it's working for all cases I know